### PR TITLE
Desktop-Version__Responsive

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Added index.HTML and style.css files with the right tags and styles.
-Created a README.md file with a detailed explanation of the project.
-Configured linters.
-Added a .gitignore file to ignore node_modules.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (SajjadAhmad14) [inline .github/workflows/linters.yml:1]: Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1876 @@
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin-left: -15px;
+    margin-bottom: 20px;
+  }
+}
+
+@media (max-width: 483px) {
```
Comment (SajjadAhmad14) [inline style.css:1866]: Please write value for min-width property in range of 992px - 1100px).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+                    <li class="bar1"></li>
+                </ul>
+                <ul class="menue_bar_desktop">
+                    <li class="nav_list"><a href="#" class="nav_link">Portofolio</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">About</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="main__wrapper">
```
Comment (SajjadAhmad14) [inline index.html:33]: Please use background images for the Headline section as decoration from the design page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    
+    <section class="grid__wrapper">
+        <div class="covers">
+            <img class="img__class" id="pic__2" src="./img/Snap.png" alt="">
+            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
```
Comment (SajjadAhmad14) [inline index.html]: The image for the first project is broken. KIndly see the screenshot below as a reference
image
.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+                </ul>
+                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+                <ul class="platform">
+                    <li class="pla__btn">html</li>
+                    <li class="pla__btn">css</li>
+                    <li class="pla__btn">javascript</li>
+                </ul>
+                <button class="see-project">See Project</button>
```
Comment (SajjadAhmad14) [inline index.html:68]: Please use a bit of small padding for the `See Project` button. KIndly see the screenshot below for reference.
btn

File: index.html [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+                    <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                    <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>  
+                </ul>
+                <button class="see-project" id="project_about">See Project</button>
+            </div>
+            <div class="performance__wrapper">
+                <div class="performance">
+                    <h3 class="languages-title">Languages</h3>
```
Comment (SajjadAhmad14) [inline index.html]: Please add a `down arrow` on the very right side of `Languages` as it can be seen in the design.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+            <h3 class="form__title">Contact me</h3>
+            <p class="form__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form class="main__form" action="https://formspree.io/f/xknkypba" method="post">
+                <input name="name" id="name" class="input__space" type="text" placeholder="Enter your Name" maxlength="30" required>
+                <input name="email" id="email" class="input__space" type="email" placeholder="Email adress" required>
+                <textarea class="text__area" name="message" id="message" cols="30" rows="10" placeholder="write your message here" maxlength="500" required></textarea>
+                <button class="input__btn" type="submit">Get it Touch</button>
+            </form>
+        </div>
+    </section>
```
Comment (SajjadAhmad14) [inline index.html:219]: Please remove extra space from the bottom of the contact form. Kindly see the screenshot below for reference.
extra

File: index.html [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+            
+        </div>
+        
+    </section>
+
+    <section class="contact-me_page">
+        <div class="contact__wrapper">
+            <h2 class="contact__header">Contact me</h2>
+            <p class="description__contact">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <form>
```
Comment (SajjadAhmad14) [inline index.html]: Please add form validations, `action`, and `post` attributes. Moreover please use `Formspree` to submit the message to user's email.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Addisalem Beidework, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @addisalemseifu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    
+    <section class="grid__wrapper">
+        <div class="covers">
+            <img class="img__class" id="pic__2" src="./img/Snap.png" alt="">
+            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
```
Comment (mke2111) [inline index.html:52]: - [ ] As previously requested, fix the image of the first project on desktop screens,
Screenshot from 2021-09-24 10-15-46

**Comment by addisalemseifu** [comment]
The broken picture is working on my machine I don't understand why it is appearing broken on the remote version.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Addisalem Beidework, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**